bugsnag-plugin-android-ndk/src/test/java/com/bugsnag/android/NativeEventDecoderTest.kt

```diff
@@ -0,0 +1,64 @@
+package com.bugsnag.android
+
+import com.bugsnag.android.ndk.NativeArch
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentCaptor
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.verify
+import org.mockito.Mockito.`when`
+import org.mockito.junit.MockitoJUnitRunner
+import java.nio.ByteBuffer
+import java.nio.ByteOrder
+
+@RunWith(MockitoJUnitRunner::class)
+class NativeEventDecoderTest {
+    private val sample32BitData =
+        this::class.java.getResourceAsStream("/arm32-droid-razr.crash_dump")!!.readBytes()
```

Comment:
Give that this test covers the 32bit crash dump, maybe it should be named accordingly?
```suggestion
class NativeEventDecoder32bitTest {
    private val crashDumpData =
        this::class.java.getResourceAsStream("/arm32-droid-razr.crash_dump")!!.readBytes()
```